db/data_migration/20171018095703_fix_govuk_pay_url.rb

```diff
@@ -0,0 +1,20 @@
+slug = "govuk-pay"
+redirect_path = "https://www.payments.service.gov.uk"
+
+document = Document.find_by(slug: slug)
+exit unless document
+
+attachment_content_ids = document.editions.flat_map do |edition|
+  edition.attachments.map(&:content_id)
+end.uniq
+
+content_ids = [document.content_id] + attachment_content_ids
+
+document.delete
```

Comment:
I don't think you should delete the document in this. I think the correct thing to do would be to unpublish it with a redirect unpublish which will achieve the same thing but leave the history of the document in Whitehall. Is there a reason you didn't do that?